core/notion.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+import requests
+from django.conf import settings
+from django.utils import timezone
+
+logger = logging.getLogger(__name__)
+
+
+def create_new_row_in_notion_table(database_id, data):
+    notion_token = settings.NOTION.get("TOKEN")
+    if not notion_token:
+        logging.warning("The notion token is not set in local environment")
```

Comment:
```suggestion
        logging.error("The notion token is not set in local environment")
```

Pour qu'on ait une alerte sentry ?

Reply:
Hmmm, en effet peut-être. 
J'ai hésité mais ca peut etre mieux